djangocms_navigation/test_utils/factories.py

```diff
@@ -13,10 +13,11 @@
 
 class UserFactory(factory.django.DjangoModelFactory):
     username = FuzzyText(length=12)
-    first_name = factory.Faker('first_name')
-    last_name = factory.Faker('last_name')
+    first_name = factory.Faker("first_name")
```

Comment:
hmmm, why all these changes from `'` to `"`? Both are correct, but generally most of the time when the strings are short like here people tend to use `'` and when they're sentences `"` is used more commonly.